plugins/hydra_ax_sweeper/tests/apps/polynomial_with_coefficients.yaml

```diff
@@ -0,0 +1,22 @@
+# @package _group_
```

Comment:
do not specify `@package` for the primary config file. (and certainly not group).